src/core/utils/default_nav_groups.ts

```diff
@@ -38,10 +38,11 @@ const defaultNavGroups = {
   all: {
     id: ALL_USE_CASE_ID,
     title: i18n.translate('core.ui.group.all.title', {
-      defaultMessage: 'Analytics (All)',
+      defaultMessage: 'Analytics (all features)',
```

Comment:
The use case name might be `Analytic` 
![image](https://github.com/user-attachments/assets/a01a794b-68d6-4d39-82bc-c4116456dc3e)

![image](https://github.com/user-attachments/assets/06a29580-6568-4071-9fa7-4cbade067d88)

Reply:
![image](https://github.com/user-attachments/assets/1ee8a858-c484-499a-815a-f908841e196c)
They are using "Analytics (all features)" in the use case card side. Let me customized the use case name here. Will confirm with UX, shall we using different name in other places.